Pipfile

```diff
@@ -26,8 +26,10 @@ sentry-sdk = "~=1.3.1"
 rq = "~=1.9.0"
 wagtail = "~=2.14.1"
 django-lti-provider = "~=0.4.4"
```

Comment:
I tried to suppress `RemovedInDjango40Warning` log warnings from `lti_provider` module, but this is a bit of a pain as it appears that we'd need to do some modifications to the log config, and as there is only a single `settings.py` right now it seemed too intrusive. It can also be done by setting `DEBUG = False` in settings, but that has larger effects than simply cleaning up log output.